sphinx/source/docs/user_guide/server.rst

```diff
@@ -1052,6 +1052,42 @@ As before, run the Bokeh server with the following command:
 
     bokeh serve myapp.py --port 5100
 
+.. _userguide_server_deployment_unix_socket:
+
+Restricting access to proxied server using unix socket
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In some cases, you might want to restrict the access to the proxied bokeh server. You can bind the bokeh server to a unix socket and use nginx or apache to proxy to a unix domain socket.
```

Comment:
We try to keep ReST file line lengths to ~80-100 (just look at the text above and below and make similar)